src/filesystem/implementation.js

```diff
@@ -2384,6 +2403,7 @@ module.exports = {
   close: close,
   mknod: mknod,
   mkdir: mkdir,
+  mkdtemp: mkdtemp,
```

Comment:
We need this to happen in https://github.com/filerjs/filer/blob/fe41a429dd3a5a57caa0df38630e53427092832b/src/filesystem/interface.js#L267-L306 too.